install/migrations/update_10.0.x_to_11.0.0/configs.php

```diff
@@ -43,6 +43,7 @@
     'set_solution_tech' => '0',
     'is_demo_dashboards' => '0',
     'planned_task_state' => '1',
+    'plugins_execution_mode' => 'on', // PLUGIN::EXECUTION_MODE_ON
```

Comment:
Should be added too in the empty data php file so it can be set for new installations.